cohortextractor/cohortextractor.py

```diff
@@ -738,10 +737,6 @@ def main(args=None):
         help="Do not regenerate data if output file already exists",
         action="store_true",
     )
-    generate_cohort_parser.add_argument(
-        "--with-end-date-fix",
-        action="store_true",
-    )
```

Comment:
Is it worth leaving this in as a no-op, at least temporarily, so that studies using it don't break immediately. We could do a github search for people using it, get them all to remove it, and then remove the flag from here later.

Reply:
Very good point.  Have reinstated.